pkg/actors/actors.go

```diff
@@ -40,19 +40,19 @@ var log = logger.NewLogger("dapr.runtime.actor")
 
 // Actors allow calling into virtual actors as well as actor state management
 type Actors interface {
-	Call(req *CallRequest) (*CallResponse, error)
+	Call(ctx context.Context, req *CallRequest) (*CallResponse, error)
 	Init() error
-	GetState(req *GetStateRequest) (*StateResponse, error)
-	SaveState(req *SaveStateRequest) error
-	DeleteState(req *DeleteStateRequest) error
-	TransactionalStateOperation(req *TransactionalRequest) error
-	GetReminder(req *GetReminderRequest) (*Reminder, error)
-	CreateReminder(req *CreateReminderRequest) error
-	DeleteReminder(req *DeleteReminderRequest) error
-	CreateTimer(req *CreateTimerRequest) error
-	DeleteTimer(req *DeleteTimerRequest) error
-	IsActorHosted(req *ActorHostedRequest) bool
-	GetActiveActorsCount() []ActiveActorsCount
+	GetState(ctx context.Context, req *GetStateRequest) (*StateResponse, error)
+	SaveState(ctx context.Context, req *SaveStateRequest) error
```

Comment:
SaveState and DeleteState can be deleted completed and thus dont need these changes as actors only support TransactionalStateOperation.

Reply:
> SaveState and DeleteState can be deleted completed and thus dont need these changes as actors only support TransactionalStateOperation.

I will delete these 2 APIs in separate issue/PR and update this PR.